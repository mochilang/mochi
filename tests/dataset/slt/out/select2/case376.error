expect condition failed at :243: expect flatResult == [-3, -4, 1, 1, 1, 1300, 134, 137, 1390, 184, 1985, 2, 2046, 222, 2728, 333, 333, 364]
call graph: main
stack trace:
  main at :243
    expect flatResult == [-3, -4, 1, 1, 1, 1300, 134, 137, 1390, 184, 1985, 2, 2046, 222, 2728, 333, 333, 364]

got: [1, -4, 333, 134, 1985, 1300, 1, 2, 222, 137, 2046, 1390, 1, -3, 333, 184, 2728, 364]
