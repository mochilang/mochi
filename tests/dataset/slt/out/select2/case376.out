-3
-4
1
1
1
1300
134
137
1390
184
1985
2
2046
222
2728
333
333
364
